mapdata/WarcraftLegacies/AbilityData/Skin/1161900097.json

```diff
@@ -39,7 +39,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Gives -2 armor to nearby hostile units."
+      "Value": "Gives -4 armor to nearby hostile units."
```

Comment:
In keeping with vanilla tooltip style, we shouldn't use negative numbers to describe debuffs. "Reduces the armor of nearby hostile units by 4."

Reply:
```suggestion
      "Value": "Reduces the armor of hostile units in a radius of <A0AE,Area1> by 4."
```